### ISSUE TITLE
How to unmap default mappings for Telescope plugin?

### ISSUE THREAD

**Issue by LinuxIsCool** [siduck76/NvChad#2825]
How can I unmap these keys?
https://github.com/NvChad/NvChad/blob/156eeef8ecae812acd79d5eb58070d734a6202c6/lua/nvchad/mappings.lua#L66

In previous nvchad I had in `mappings.lua`:
```lua
---@type MappingsTable
local M = {}

M.disabled = {
  n = {
    ["<leader>f"] = "",
    ["<leader>ff"] = "",
    ["<leader>fg"] = "",
    ["<leader>fb"] = "",
    ["<leader>fh"] = "",
    ["<leader>fa"] = "",
    ["<leader>fm"] = "",
    ["<leader>fo"] = "",
    ["<leader>fw"] = "",
    ["<leader>fz"] = "",
  },
}
return M

```

Now I'm trying in `mappings.lua`:
```lua
require "nvchad.mappings"

-- Disable mappings
local nomap = vim.keymap.del

nomap("n", "<leader>f")
nomap("n", "<leader>fa")
nomap("n", "<leader>fb")
nomap("n", "<leader>ff")
nomap("n", "<leader>fh")
nomap("n", "<leader>fm")
nomap("n", "<leader>fo")
nomap("n", "<leader>fw")
nomap("n", "<leader>fz")

```

But getting error: 

```
Error executing vim.schedule lua callback: vim/keymap.lua:0: E31: No such mapping
stack traceback:
        [C]: in function 'nvim_del_keymap'
        vim/keymap.lua: in function 'nomap'
        /home/ygg/.config/nvim/lua/mappings.lua:6: in main chunk
        [C]: in function 'require'
        /home/ygg/.config/nvim/init.lua:38: in function </home/ygg/.config/nvim/init.l
ua:37>
```

**Comment by LinuxIsCool** [comment]
Furthermore, how can I simply disable Telescope?

**Comment by siduck** [comment]
check the list of mappins in nvchad first, there is no such mapping as leader f

**Comment by LinuxIsCool** [comment]
I don't understand your answer. I linked directly to the mapping. 

It's in `NvChad/lua/nvchad/mappings.lua`

And you can see it with whichkey as well.

**Comment by siduck** [comment]
![image](https://github.com/NvChad/NvChad/assets/59060246/ab5e7f40-e2ab-4b8d-a27d-89715faed68b)

there is no leader f here?

**Comment by LinuxIsCool** [comment]
I would like to unmap \<leader\>f as a prefix, thus I would like to unmap all those mappings that begin with \<leader\>f.

**Comment by siduck** [comment]
Vim.keymap.Del has nothing to do with prefixes , its which key stuff
